### PR TITLE
bugfix_value_type_of_dataschema

### PR DESCRIPTION
**Pull request by timmyb32r**
it's improved copy of #985 - I made typo and couldn't fix previous branch

### CODE REVIEW

**Comment by duglin** [comment]
Sorry for the delay - and thanks for the explanation.
/LGTM

**Review by duglin — DISMISSED** [review]

**Comment by duglin** [comment]
ping @embano1 and @lionelvillard for another set of eyes

**Review by embano1 — COMMENTED** [review]
Just some minor things/question

**Review by embano1 — APPROVED** [review]
LGTM

File: binding/format/protobuf/v2/protobuf.go [hunk 1]
```diff
@@ -251,8 +256,8 @@ func FromProto(container *pb.CloudEvent) (*event.Event, error) {
 			vs, _ := v.(string)
 			e.SetDataContentType(vs)
 		case dataschema:
-			vs, _ := v.(string)
-			e.SetDataSchema(vs)
+			vs, _ := v.(types.URI)
```
[the remark this answers is not in the document]
Comment (timmyb32r) [reply binding/format/protobuf/v2/protobuf.go:259]: good question. I've checked:
`vs, _ := v.(types.URI)` will return default constructed `types.URI{}`, and further called mechod .String() will return an empty string

**Comment by embano1** [comment]
Can you please rebase onto `main` and also sign your commits or whatever is causing the DCO to fail?

**Comment by timmyb32r** [comment]
@duglin @embano1 what we should also do to merge it? Idk why these last checks (Build/CloudEvents/Unit Test) wont pass :(

**Review by embano1 — APPROVED** [review]

**Pull request merged**